features/managing_invoices/seeing_invoices_when_items_has_promitions_applied.feature

```diff
@@ -0,0 +1,24 @@
+@managing_invoices @ui
+Feature: Seeing information about an invoice with items has promotions applied
+    In order to see details of a specific invoice
+    As an Administrator
+    I want to be able to view basic information about an invoice
```

Comment:
`In order to see details (...) I want to be able to view basic information` 😄 I know it's technically correct, but I would focus more on what we try to show in each feature (in this one - show promotions applied in the invoice)